util/argo/diff/diff.go

```diff
@@ -213,6 +234,9 @@ func (c *diffConfig) Validate() error {
 			return fmt.Errorf("%s: StateCache must be set when retrieving from cache", msg)
 		}
 	}
+	if c.serverSideDiff && c.resourceOperations == nil {
+		return fmt.Errorf("%s: restConfig must be set when using server side diff", msg)
```

Comment:
Do you mean "resourceOperations" must be set? Or does the lack of resourceOperations imply that some restConfig was not set?

Reply:
fixed